electrum/gui/qt/three_keys_windows.py

```diff
@@ -301,7 +317,7 @@ def do_pay(self):
             return
 
         keypair = None
-        if self.tx_type_combo.currentIndex() == self.TX_TYPES['fast']:
+        if self.tx_type_combo.currentIndex() == self.TX_TYPES.Secure_Fast:
```

Comment:
Lets uniform Enum type calling. We have `self.TX_TYPES['Secure']` so it will be better to change into `self.TX_TYPES['Secure_Fast']`